ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-multicast.j2

```diff
@@ -23,6 +23,20 @@
 - Software forwarding by the Linux kernel
 {%     elif router_multicast.ipv4.software_forwarding is arista.avd.defined('sfe') %}
 - Software forwarding by the Software Forwarding Engine (SFE)
+{%     endif %}
+{%     if router_multicast.rpf.routes is arista.avd.defined %}
+
+### IP Router Multicast RPF Routes
+
+| Source Prefix | Next Hop IP | Next Hop Interface | Administrative Distance |
+| ------------- | ----------- | ------------------ | ----------------------- |
+{%         for rpf_route in router_multicast.ipv4.rpf.routes %}
+{%             set nexthop_ip = rpf_route.nexthop_ip | arista.avd.default('-') %}
+{%             set nexthop_interface = rpf_route.interface | arista.avd.default('-') %}
+{%             set distance = static_rorpf_routeute.distance | arista.avd.default('1') %}
+| {{ rpf_route.subnet }} | {{ nexthop_ip }} | {{ nexthop_interface }} | {{ distance }} |
+{%         endfor %}
+
```

Comment:
We don't require this blank line. The blank lines are added at the top of block(like line28, line42 after an if check)

Reply:
good catch, my mistake